uiServices/services.go

```diff
@@ -0,0 +1,89 @@
+package services
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/Sirupsen/logrus"
+	"github.com/xtracdev/automated-perf-test/perfTestUtils"
+	"net/http"
+	"os"
+	"strings"
+	"fmt"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+func configsHandler(rw http.ResponseWriter, req *http.Request) {
+
+		configPathDir := req.Header.Get("configPathDir")
+		//error check to ensure file path ends with "\"
+		if !strings.HasSuffix(configPathDir, "/") {
+			configPathDir = configPathDir + "/"
+		}
+
+
+		config := perfTestUtils.Config{}
+		buf := new(bytes.Buffer)
+		buf.ReadFrom(req.Body)
+		defer req.Body.Close()
+
+		err := json.Unmarshal(buf.Bytes(), &config)
+
+	if err != nil {
+		logrus.Error("Failed to unmarshall json body", err)
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	if FilePathExist(configPathDir) == false {
+		logrus.Error("File path does not exist", err)
+		rw.WriteHeader(http.StatusNotFound)
```

Comment:
should be 400 here. also, consider returning a response body with the invalid field and what went wrong.